mlrun/model.py

```diff
@@ -508,12 +586,19 @@ def __init__(
         self.scrape_metrics = scrape_metrics
         self.allow_empty_resources = allow_empty_resources
 
-    def to_dict(self, fields=None, exclude=None):
-        struct = super().to_dict(fields, exclude=["handler"])
+    def to_dict(self, fields: list = None, exclude: list = None, strip: bool = False):
+        struct = super().to_dict(fields, exclude=["handler"], strip=strip)
```

Comment:
Do we need the `exclude` param? isn't the whole point to use the `_fields_to_strip` instead?

Reply:
Mainly if you want to apply something manually